src/transport/tests/LoopbackTransportManager.h

```diff
@@ -98,6 +98,8 @@ class LoopbackTransportManager
         }
     }
 
+    void StepDrainAndServiceIO() { mIOContext.DriveIO(); }
```

Comment:
This should probably be called "DeliverOneMessage" or something like that.  Assuming that's what it actually does?